README.md

```diff
@@ -231,16 +231,16 @@ before_script:
 script:
   - cobalt build
 
-after_success: |
-  [ $TRAVIS_BRANCH = master ] &&
-  [ $TRAVIS_PULL_REQUEST = false ] &&
-  cobalt import &&
-  git config user.name "Cobalt Site Deployer" &&
-  git config user.email "name@example.com" &&
-  git push -fq https://${GH_TOKEN}@github.com/${TRAVIS_REPO_SLUG}.git gh-pages
+deploy:
+  provider: pages
+  skip_cleanup: true
+  github_token: $GH_TOKEN
+  local_dir: build
+  on:
+    branch: master
```

Comment:
So this will automatically push the results of `cobalt build` into the appropriate branch?

Have a link where I can learn  more?

Reply:
Oh sorry. I put a link to the documentation into the commit message but forgot to include it in the PR text, here it is: https://docs.travis-ci.com/user/deployment/pages/

It pushed into `gh-pages` by default but this can be configured.